lib/rules/func-call-spacing.js

```diff
@@ -86,9 +86,9 @@ module.exports = {
                 return;
             }
 
-            const hasWhitespace = sourceCode.isSpaceBetweenTokens(prevToken, parenToken);
-            const hasNewline = hasWhitespace &&
-                /\n/.test(text.slice(prevToken.range[1], parenToken.range[0]).replace(/\/\*.*?\*\//g, ""));
+            const textBetweenTokens = text.slice(prevToken.range[1], parenToken.range[0]).replace(/\/\*.*?\*\//g, "");
+            const hasWhitespace = /\s/.test(textBetweenTokens);
+            const hasNewline = hasWhitespace && /[\n\r\u2028\u2029]/.test(textBetweenTokens);
```

Comment:
> I think it would be a very good idea to centralize part of the code in question to make it reusable, although it should be probably done in a different issue.

:+1: Agreed. Will you create an issue to keep track of that?